testbed/testbed/mock_backend_test.go

```diff
@@ -70,7 +70,7 @@ func WaitFor(t *testing.T, cond func() bool, errMsg ...interface{}) bool {
 			return true
 		}
 
-		if time.Since(startTime) > time.Second*5 {
+		if time.Since(startTime) > time.Second*10 {
```

Comment:
this feels like it should be a parameter but it can wait until a real case usage of different values

Reply:
Sure, we can add it if needed. My past experience has been that it is needed very rarely and it is easier to make it a global settings to avoid specifying it for every waitfor.